buildchow/etc/todirformat.sh

```diff
@@ -0,0 +1,14 @@
+FILES=$1/*_$2
```

Comment:
What are $1 and $2 supposed to contain? Please document usage or provide a usage() method which runs if no params are specified. Example:

https://github.com/jbosstools/jbosstools-build-ci/blob/master/util/installFromTarget.sh